sources/CascadiaCode-Regular.ufo/lib.plist

```diff
@@ -13,9 +13,13 @@
     <key>GSOffsetVertical</key>
     <integer>110</integer>
     <key>com.schriftgestaltung.appVersion</key>
-    <string>1272</string>
+    <string>1274</string>
     <key>com.schriftgestaltung.customParameter.GSFont.DisplayStrings</key>
-    <string></string>
+    <array>
+      <string>/ereversed-cy</string>
+    </array>
+    <key>com.schriftgestaltung.customParameter.GSFont.Make morx table</key>
```

Comment:
do we definitely want the `morx` table? this looks like it's setting "make morx table" to `1`

Reply:
Derp. No. That setting actually doesn't do anything (apparently you have to *have* a <morx> table in order for it to work), and I forgot to remove it.